### PR TITLE
Getting started toc & subsections

### PR DESCRIPTION
**Pull request by juanmaguitar**
Also some linter fixing

### CODE REVIEW

File: docs/getting-started/README.md [hunk 1]
```diff
@@ -2,17 +2,33 @@
...
+- [Get up and running](#get-up-and-running)
+
+<!-- tocstop -->
 
 ## Installation requirements
 
 To get started with Frontity you will need:
 
-* **A WordPress installation** - This can be hosted locally, on a web-server, or you can also use a site hosted on wordpress.com.
+* **A WordPress installation** - This can be hosted locally, on a web-server, or you can also use a site hosted on wordpress.com
```
Comment (mburridge) [inline docs/getting-started/README.md:27]: Why are you removing the full stop from the end of the sentence here?

File: docs/getting-started/README.md [hunk 2]
```diff
@@ -2,17 +2,33 @@
...
 
   However, when you spin up your first Frontity project you will initially be connected to our test site so you won't be without content when you first try Frontity
 
-* **Node.js installed in your computer** - If you don't already have it you can get **Node.js** from [the official site](https://nodejs.org/). This will also install **npm** and **npx** along with Node.js.
+* **Node.js installed in your computer** - If you don't already have it you can get **Node.js** from [the official site](https://nodejs.org/). This will also install **npm** and **npx** along with Node.js
```
Comment (mburridge) [inline docs/getting-started/README.md:31]: And here?

**Comment by juanmaguitar** [comment]
@mburridge These periods are being removed automatically when doing `npm texlint --fix <some-file>` because of this rule → https://github.com/frontity/docs/blob/e942feb10ccfbeb0884e0b86d3fee58569b5366f/package.json#L68 

It may be very subtle, but from my point of view, I think removing these periods make this items to be perceived more like a part of a list.

But anyway, we can disable it if you want. It is not clear to me what's the right way of writing this.

**Comment by mburridge** [comment]
> These periods are being removed automatically when doing npm texlint --fix <some-file>

Oh, I see. It's not immediately clear that this is a list as both the list items have a paragraph following them which doesn't appear to be a list item. 

I'd say disable this rule so we can choose whether we terminate a list item with a period or not.

**Pull request merged**